frontend/src/components/selectInput/SelectInput.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import style from "./selectInput.module.scss";
+
+interface SelectInputProps {
+    label: string;
+    onChange: () => void;
+    value: string;
+    id: string;
+}
+
+const categoryOptions = ["PNG", "GIF", "WEBP", "MP4", "MP3"];
+
+export const SelectInput = ({ label, onChange, value, id }: SelectInputProps) => {
+    return (
+        <div className={style.selectContainer}>
+            <label className={style.label}>{label}</label>
```

Comment:
issue: the label is missing the `htmlFor` for linking it to the select input.

It might not be needed, but i'd suggest double checking.